### DISCUSSION TITLE
NPM inside LAN without internet access

### DISCUSSION THREAD

**Discussion by rmoggia** [NginxProxyManager/nginx-proxy-manager#2571] [Q&A]
Hello. I have to run NGINX proxy manager within a LAN without internet access. NPM must receive https requests to validate them and redirect them to hosts that are located behind it. Obviously I can't use letsencrypt. But I have the server certificates available and the npm server name is resolved from an internal DNS. is it possible to use npm in this situation?
Thanks

**Comment by dinccey** [comment]
I use it like that with my own certificates, just import custom cert into NPM and use it for your sites.
